tests/x11regressions/gnote/gnote_search_body.pm

```diff
@@ -23,6 +23,8 @@ sub run {
     send_key_until_needlematch 'gnote-start-here-matched', 'down', 5;
     send_key "ret";
     send_key "ctrl-f";
+    # The gnote interface is slow. So we can't start immediatly searching. We need to wait
```

Comment:
[DRY](https://en.wikipedia.org/wiki/Don't_repeat_yourself). You are calling the same code multiple times, please [extract method](http://refactoring.com/catalog/extractMethod.html).